### PR TITLE
Improve initiative description

### PR DESCRIPTION
**Pull request by MattiSG**
Capture d’écran 2019-05-20 à 20 48 27

### CODE REVIEW

File: _pages/index.html [hunk 1]
```diff
@@ -28,16 +28,22 @@
...
-					</p>
+					<div class="column is-10 is-offset-1">
+						<h1 class="title is-spaced">
+							Tools to fight against disinformation
+						</h1>
+						<h2 class="subtitle">
+							We reinforce the democratic antibodies that are already present in our societies.
+						</h2>
+						<p class="description">
+							Information manipulation tries to disrupt the social cohesion and electoral processes that keep democratic societies together.
```
Comment (BerylBogui) [inline _pages/index.html]: Grammatically an active form is incorrect, use passive:
Ex: Information manipulation aims at disrupting  / is meant to disrupt

File: _pages/index.html [hunk 2]
```diff
@@ -28,16 +28,22 @@
...
+						</p>
+						<p class="description">
+							We believe that there are already strong forces against disinformation campaigns in them, but that they are too isolated to confront well-organised actors. And thus that collaboration, cooperation and mutualisation are the key towards effective action.
```
Comment (BerylBogui) [inline _pages/index.html]: Simplified for clarity:
We believe that they already encompass strong forces against disinformation, but...

Comment (BerylBogui) [inline _pages/index.html]: . Thus, collaboration ...
(It is still very understandable that the second sentence is part of the same reasoning ;) There's no need for a double proposition. )

File: _pages/index.html [hunk 3]
```diff
@@ -28,16 +28,22 @@
...
 				<div class="columns">
-					<div class="column is-8 is-offset-2">
-					<h1 class="title is-spaced">
-						Fight against disinformation
-					</h1>
-					<h2 class="subtitle">
-						We aim at reinforcing the democratic antibodies that are present in our societies.
-					</h2>
-					<p class="description">
-						Under the ageis of the French Ambassador for Digital Affairs, our team focuses on collecting best practices and developing tools to counter disinformation, its spread and its consequences. We believe that collaboration, cooperation and mutualisation are the key towards effective action against disinformation.
```
Comment (BerylBogui) [inline _pages/index.html]: * aegis
Typo, my bad

**Comment by BerylBogui** [comment]
Made the minor changes suggested in the review, to simplify, clarify but not affect the meaning of the text :)

File: _pages/index.html (hunk 1 above)
Comment (MattiSG) [reply _pages/index.html]: Yup I agree, I changed this before increasing the width of the container, had to make that change for it to fit on a single line… But now it does fit AND is correct 🙂

File: _pages/index.html (hunk 2 above)
Comment (MattiSG) [reply _pages/index.html]: You're right, neither phrasings are very clear when read. I'll change that to repeating “societies”.

Comment (MattiSG) [reply _pages/index.html]: This rephrasing unfortunately becomes assertive, while I wanted to convey the idea that it was still our belief.

**Pull request merged**